apimatic_core/request_builder.py

```diff
@@ -184,8 +179,6 @@ def process_body_params(self):
             self.add_additional_form_params()
             return ApiHelper.form_encode_parameters(self._form_params, self._array_serialization_format)
         elif self._body_param is not None and self._body_serializer:
-            if self._should_wrap_body_param:
```

Comment:
Yes we have removed it because this flag was used for old OneOf/AnyOf design. Since we have moved to the new design therefore this because unused now. Also, I have removed its test coverage as well.